src/utils/user-helper.ts

```diff
@@ -0,0 +1,2 @@
+export const isBuddy = (role: string) => role === 'BUDDY';
```

Comment:
Isn't possible to pick this roles from the generated schema ENUM?

Reply:
Then we need to add similar schema-types from here `buddy-app/server/src/generated/schema-types.ts` file to `src/types` folder for example